languages/elixir/exercises/concept/rpg-character-sheet/.docs/instructions.md

```diff
@@ -0,0 +1,61 @@
+You and your friends love to play pen-and-paper role-playing games, but you noticed that it's difficult to get new people to join your team. They often struggle with character creation. They don't know where to start. To help new players out, you decided to write a small program that will guide them through the process.
+
```

Comment:
I hope this makes sense 🙈 I never played any pen-and-paper RPGs.

Reply:
I don't play any for this reason, I think its a great premise :tada:

Reply:
The pain is real.